app/elements/pw-shell.html

```diff
@@ -445,6 +448,7 @@
     },
 
     _handleSearchResponse: function(event) {
+      var i;  // the linter which may not be named has issues.
```

Comment:
A story: I tried to make the comment be "// jshint has issues with scoping" and then `jshint` tried to interpret "has issues with scoping" as weird config parameters and anyway, `jshint` is Voldemort now, this is our darkest timeline.